src/ripple/rpc/impl/RPCHelpers.h

```diff
@@ -212,26 +212,44 @@ extern beast::SemanticVersion const lastVersion;
  * version 1.
  *
  * Command line Requests use ApiMaximumSupportedVersion.
+ *
+ * ApiMaximumSupportedVersion is not const for testing
+ * experimental API versions that are not enabled for public
+ * consumption. In unit tests, ApiMaximumSupportedVersion
+ * can be promoted to ApiExperimentalVersion.
+ *
+ * @note ApiMaximumSupportedVersion can only be changed
+ * by unit tests.
  */
 
-constexpr unsigned int APIInvalidVersion = 0;
-constexpr unsigned int APIVersionIfUnspecified = 1;
+extern unsigned int ApiMaximumSupportedVersion;
+constexpr unsigned int ApiInvalidVersion = 0;
+constexpr unsigned int ApiVersionIfUnspecified = 1;
 constexpr unsigned int ApiMinimumSupportedVersion = 1;
-constexpr unsigned int ApiMaximumSupportedVersion = 1;
-constexpr unsigned int APINumberVersionSupported =
-    ApiMaximumSupportedVersion - ApiMinimumSupportedVersion + 1;
+constexpr unsigned int ApiExperimentalVersion = 2;
+constexpr unsigned int ApiNumberVersionSupported =
```

Comment:
Thanks. They are changed to start with a lower case `api`. 
They are not changed to functions, because `apiMaximumSupportedVersion` is changed back to `constexpr`.